src/visualization/plot_recipes.jl

```diff
@@ -58,6 +58,23 @@ struct PlotData2D{Coordinates, Data, VariableNames, Vertices} <: AbstractPlotDat
   orientation_y::Int
 end
 
+"""
+    DGMultiPlotData{Dim, uType, VariableNames}
+
+Holds data for creating 2D plots of multiple solution variables for DGMulti solvers.
```

Comment:
Does it only work for `DGMulti` or will it also be possible to use it for other unstructured/curved mesh-solver combinations? (which would be awesome!)

Reply:
With pretty minor changes, this is usable for all mesh types (same as Makie). I can add those in this PR if that's of interest too.

Reply:
That would be awesome :+1: